server/service/osquery_utils/queries.go

```diff
@@ -450,7 +450,7 @@ SELECT
   'python_packages' AS source
 FROM python_packages;
 `,
-	Platforms:  []string{"linux", "rhel", "ubuntu", "centos"},
+	Platforms:  []string{"linux", "rhel", "ubuntu", "centos", "debian"},
```

Comment:
Hm, we should make sure to have `debian` and `centos` everywhere we want to reference linux. I see for instance in http://github.com/fleetdm/fleet/blob/142006cbdd52f4efeefed3144263f9c1b4192a04/server/datastore/mysql/hosts.go#L270-L270 we do have debian, but not centos.

Reply:
re: http://github.com/fleetdm/fleet/blob/142006cbdd52f4efeefed3144263f9c1b4192a04/server/datastore/mysql/hosts.go#L270-L270

We probably didn't catch this during manual testing because e.g. the CentOS simulated hosts identify themselves as `rhel`. Adding `centos` there just in case of older versions of OS or osquery makes sense and shouldn't cause issues.

Reply:
Also, is `linux` a possible value for `Host.Platform`? If so, we should add it too.

Reply:
Here's https://github.com/fleetdm/fleet/pull/3217

We can merge this PR and work on #3217 later (after the release).